core/src/main/java/tc/oc/pgm/community/commands/ModerationCommands.java

```diff
@@ -281,6 +307,290 @@ public void ipBan(
+        sender.sendMessage(
+            new PersonalizedTranslatable(
+                    "moderation.commands.noAlts", targetMP.getStyledName(NameStyle.FANCY))
+                .getPersonalizedText()
+                .color(ChatColor.RED));
+      } else {
+        sender.sendMessage(formatAltAccountList(targetMP, alts));
+      }
+    } else {
+      List<Component> altAccounts = Lists.newArrayList();
+      List<MatchPlayer> accountedFor = Lists.newArrayList();
+
+      for (Player player : Bukkit.getOnlinePlayers()) {
+        MatchPlayer targetMP = manager.getPlayer(player);
+        List<MatchPlayer> alts = getAltAccounts(player, manager);
+
+        if (alts.isEmpty() || accountedFor.contains(targetMP)) {
+          continue;
+        } else {
+          altAccounts.add(formatAltAccountList(targetMP, alts));
+          accountedFor.add(targetMP);
+          accountedFor.addAll(alts);
+        }
+      }
+
+      int perPage = 8;
+      int pages = (altAccounts.size() + perPage - 1) / perPage;
+
+      Component pageNum =
+          new PersonalizedTranslatable(
+                  "command.paginatedResult.page",
+                  new PersonalizedText(Integer.toString(page)).color(ChatColor.DARK_AQUA),
+                  new PersonalizedText(Integer.toString(pages)).color(ChatColor.DARK_AQUA))
+              .getPersonalizedText()
+              .color(ChatColor.GRAY);
+
+      Component headerText =
+          new PersonalizedTranslatable("moderation.alts.header")
+              .getPersonalizedText()
+              .color(ChatColor.DARK_AQUA);
+
+      Component header =
+          new PersonalizedText(
+              headerText,
+              new PersonalizedText(" (").color(ChatColor.GRAY),
+              new PersonalizedText(Integer.toString(altAccounts.size())).color(ChatColor.DARK_AQUA),
+              new PersonalizedText(") » Page ").color(ChatColor.GRAY),
+              pageNum);
+
+      Component formattedHeader =
+          new PersonalizedText(
+              ComponentUtils.horizontalLineHeading(
+                  ComponentRenderers.toLegacyText(header, sender), ChatColor.BLUE));
+
+      new PrettyPaginatedComponentResults<Component>(formattedHeader, perPage) {
+        @Override
+        public Component format(Component data, int index) {
+          return data;
+        }
+
+        @Override
+        public Component formatEmpty() throws CommandException {
+          throw new CommandException("No alternate accounts found!");
+        }
+      }.display(audience, altAccounts, page);
+    }
+  }
+
+  @Command(
+      aliases = {"baninfo", "lookup", "l"},
+      usage = "[player] -u (if input is UUID)",
+      desc = "Lookup baninfo about a player",
+      perms = Permissions.STAFF)
+  public void banInfo(CommandSender sender, String target, @Switch('u') boolean uuid)
+      throws CommandException {
+    BanEntry ban = Bukkit.getBanList(BanList.Type.NAME).getBanEntry(target);
+
+    if (ban != null) {
```

Comment:
Switch this one arround to:
 ```
if (ban == null ||
     (ban.getExpiration() != null && ban.getExpiration.toInstant().isBefore(Instant.now())) { 
    throw CommandEx...;
}
```
It will make the rest of the command not be indented by one and probably easier to read